app/__main__.py

```diff
@@ -11,24 +12,28 @@
 parser = argparse.ArgumentParser(
     prog=__package__,
     description="preview raw data as an image of chosen format")
-parser.add_argument("FILE_PATH", help="file containing raw image data")
+parser.add_argument("-f",
+                    "--FILE_PATH",
+                    help="file containing raw image data",
+                    default=None)
 parser.add_argument("-c",
                     "--color_format",
                     choices=AVAILABLE_FORMATS.keys(),
                     default=list(AVAILABLE_FORMATS.keys())[0],
                     help="target color format (default: %(default)s)")
 parser.add_argument("-r",
                     "--resolution",
```

Comment:
Can the resolution here be changed to "width", considering the user no longer gives the height of image? This goes on as the argument "resulotion" is passed throughout the app.